signer-service/src/constants/constants.js

```diff
@@ -9,12 +9,25 @@ const SUBSIDY_MINIMUM_RATIO_FUND_UNITS = '10'; // 10 Subsidies considering maxim
 const MOONBEAM_RECEIVER_CONTRACT_ADDRESS = '0x0004446021fe650c15fb0b2e046b39130e3bfe36';
 const STELLAR_EPHEMERAL_STARTING_BALANCE_UNITS = '2.5'; // Amount to send to the new stellar ephemeral account created
 const PENDULUM_EPHEMERAL_STARTING_BALANCE_UNITS = '0.1'; // Amount to send to the new pendulum ephemeral account created
+const DEFAULT_LOGIN_EXPIRATION_TIME_HOURS = 7 * 24;
+const VALID_SIWE_DOMAINS = [
+  'localhost:5173',
+  'polygon-prototype-staging--pendulum-pay.netlify.app',
+  'app.vortexfinance.co',
+];
+const VALID_SIWE_CHAINS = [137]; // 137: Polygon
+const VALID_SIWE_LOGIN_ORIGINS = [
+  'http://localhost:5173',
+  'https://polygon-prototype-staging--pendulum-pay.netlify.app',
+  'https://app.vortexfinance.co',
+];
 
 require('dotenv').config();
 
 const PENDULUM_FUNDING_SEED = process.env.PENDULUM_FUNDING_SEED;
 const FUNDING_SECRET = process.env.FUNDING_SECRET;
 const MOONBEAM_EXECUTOR_PRIVATE_KEY = process.env.MOONBEAM_EXECUTOR_PRIVATE_KEY;
+const SEP10_MASTER_SECRET = FUNDING_SECRET;
```

Comment:
I think so, although I tried to offramp only with it and it didn't login immediately. Since Anclap is still doing testing (I believe with this account), and the code should not allow to sign a transaction without a memo for them, we can leave it like this for now.

Reply:
Okay!